### ISSUE TITLE
[Bug]: FluentUI DatePicker Accessibility Issue aria-required-children

### ISSUE THREAD

**Issue by mzavala722** [microsoft/fluentui#27589]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
Package version: @fluentui/react: 8.93.1
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://developer.microsoft.com/en-us/fluentui#/controls/web/datepicker

## Repro Steps
- Navigate to the documentation for the DatePicker component: https://developer.microsoft.com/en-us/fluentui#/controls/web/datepicker
- Open a DatePicker so that the calendar is visible
- Run the Accessibility Insights for Web fast pass.

## Actual Behavior
The FluentUI DatePicker component is currently failing the Accessibility Insights for Web fast pass. See screenshots and attached logs.
![image](https://user-images.githubusercontent.com/130993509/232490861-3ce9d0e5-551f-4407-a8ab-06e91dc32e20.png)
![image](https://user-images.githubusercontent.com/130993509/232491261-43b89f65-7f20-484a-a8ed-1d11188f3afc.png)

## Expected Behavior
The DatePicker component should pass the Accessibility Insights for Web fast pass without any accessibility issues.

### Logs

```shell
Accessibility Fast Pass Issue Log:
Title: WCAG 1.3.1: Ensures elements with an ARIA role that require child roles contain them (table)
Tags: Accessibility, WCAG 1.3.1, aria-required-children

Issue: Ensures elements with an ARIA role that require child roles contain them (aria-required-children - https://accessibilityinsights.io/info-examples/web/aria-required-children)

Target application: Fluent UI - Controls - React - DatePicker - https://developer.microsoft.com/en-us/fluentui#/controls/web/datepicker

Element path: table

Snippet: <table class="table-413" aria-multiselectable="false" aria-labelledby="id__526" aria-activedescendant="id__527" role="grid">

How to fix: 
Fix any of the following:
  Element has children which are not allowed (see related nodes)
  Element has no aria-busy="true" attribute

Environment: Microsoft Edge version 112.0.1722.39

====

This accessibility issue was found using Accessibility Insights for Web 2.38.1 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
```

### Requested priority

Blocking

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by smhigley** [comment]
This is a false positive from axe-core, which is detailed in this issue: https://github.com/dequelabs/axe-core/issues/3850. They already have a fix, which should be available in the next release (4.8, I believe).
